src/components/FindMap.jsx

```diff
@@ -0,0 +1,51 @@
+import { styled } from 'styled-components';
+
+import LogoIcon from '../assets/images/logo-design-icon.svg';
+
+import LinkComponent from './LinkComponent.jsx';
+
+const FindMap = () => {
+  return (
+    <FindMapContainer>
+      <ContentBox>
+        <IconConatiner>
+          <img src={LogoIcon} alt="logo-icon" />
+          <IconText>여기 가볼래요 !</IconText>
+        </IconConatiner>
+        <LinkComponent text="카카오맵으로 길찾기" color="#007DFF" />
```

Comment:
여기에 달 리뷰는 아니지만 남겨보자면, <button> 이나 <a> 를 기반으로 하는 <Button> 이나 <Link> 컴퍼넌트는 그 메인 내용물은 children으로 넣는게 일반적이에요. 이 경우는 아래처럼이 일반적이겠군요.

```suggestion
        <LinkComponent color="#007DFF">카카오맵으로 길찾기</LinkComponent>
```